### PR TITLE
feat: add lambda-ddb-mysql-etl-pipeline python service

### PR DESCRIPTION
**Pull request by benfarr**
Fixes # There are no fixes. Simply a sample project contribution for others to use. 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by benfarr** [comment]
@selvathiruarul

**Comment by benfarr** [comment]
@garnaat Hello, Sir. Not quite sure what's required for next steps to get this approved?

File: python/lambda-ddb-mysql-etl-pipeline/app.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+PROD_ENV={
+	"region": "<region_id>",
+	"account": "<account_id>"
+}
+
+STAGE=os.environ.get("AWS_CDK_ENV")
+
+with open(f"params-{STAGE}.json") as file_obj:
```
Comment (NGL321) [inline python/lambda-ddb-mysql-etl-pipeline/app.py]: Seems to fail here
``` shell
2020-01-18T00:52:38.5463314Z Traceback (most recent call last):
2020-01-18T00:52:38.5464150Z   File "app.py", line 19, in <module>
2020-01-18T00:52:38.5465233Z     with open(f"params-{STAGE}.json") as file_obj:
2020-01-18T00:52:38.5466320Z FileNotFoundError: [Errno 2] No such file or directory: 'params-None.json'
2020-01-18T00:52:38.6098692Z [31mSubprocess exited with error 1[39m
2020-01-18T00:52:44.7610111Z ##[error]Process completed with exit code 1
```

Comment (richardhboyd) [reply python/lambda-ddb-mysql-etl-pipeline/app.py]: for the submitter: When we have an example with a required environment variable it's generally best to add it as a context variable instead and update the cdk.json file to reflect a valid value for it.

Here is an example application that does such a thing

[Referencing a context variable](https://github.com/aws-samples/aws-cdk-examples/blob/e8d9b0427b8122aaff33b44005f4744ce9f31124/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/index.ts#L15-L17)

[Where the variable is set](https://github.com/aws-samples/aws-cdk-examples/blob/e8d9b0427b8122aaff33b44005f4744ce9f31124/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/cdk.json)

File: python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+    Args:
+        event (dict, list, str, int, float, NoneType): Provides event specific data to the handler. It
+            can also be of type list, str, int, float, or NoneType.
+        context (obj): Provides methods and properties that contain invocation, function and
+            execution environment information.
+    Raises:
+        Exception: If file format is not of .json or .csv file types.
+
+    Notes:
```
Comment (NGL321) [inline python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py]: I appreciate what you are trying to do here, but having large blocks of commented out code is not a practice we want to include on our repos.

Could you find a way to include this without the commented out code? (fully implement, or scale down?)

**Comment by NGL321** [comment]
Hey @benfarr. still working on this?

**Comment by benfarr** [comment]
> Hey @benfarr. still working on this?

Hey @NGL321. Just noticing all these. Thanks for the feedback. I'll get all this cleaned up. Have some deployments at work I need to get out the door this week but will get to refactor beginning of next week most likely. Thanks for taking the time to review!

File: python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py (hunk 2 above)
Comment (benfarr) [reply python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py]: @NGL321 no docstrings in general in preferred or mainly additional notes?

**Comment by benfarr** [comment]
> Wow! This is a serious example. I appreciate the real-world scenario implementation, I think this will be a great addition to the repo.
> 
> I have requested a few changed. Unfortunately, one of them may require a decent refactoring, but I am looking forward to following the process and getting this merged in.
> 
> 😸

No problem! Happy to contribute to a fun project.

**Comment by benfarr** [comment]
@NGL321 resolved all comments and fully implemented mysql example. Should be good now.

**Comment by benfarr** [comment]
> Wonderful! The fixes look great! Thank you again for submitting such an in-depth example.
> 
> 😸

My pleasure! Anything else required of me to get this merged?

**Pull request merged**